scss/_carousel.scss

```diff
@@ -127,6 +128,15 @@
     --#{$boosted-prefix}control-bg: #{$carousel-control-icon-active-bg}; // Boosted mod
     color: $carousel-control-color;
   }
+
+  &:focus:focus-visible,
+  &:focus {
```

Comment:
I think this selector could be upgraded with a `[data-focus-visible-added]` in order to not have the focus state on click